rest-service/manager_rest/blueprints_manager.py

```diff
@@ -315,6 +321,29 @@ def execute_workflow(self, deployment_id, workflow_id,
 
         return new_execution
 
+    def _check_for_any_active_executions(self):
+        executions = [
+            e.id
+            for e in self.executions_list(is_include_system_workflows=True)
+            if e.status not in e.END_STATES
+        ]
+
+        if executions:
+            raise manager_exceptions.ExistingRunningExecutionError(
+                'You cannot start system wide execution if there are '
+                'running other executions. '
+                'Executions currently running: {0}.'
+                .format(executions))
+
+    def _check_for_active_system_wide_execution(self):
+        for e in self.executions_list(is_include_system_workflows=True):
+            if e.status not in e.END_STATES and not e.deployment_id:
+                raise manager_exceptions.ExistingRunningExecutionError(
+                    'You cannot start execution if there is running system '
```

Comment:
Maybe "You cannot start an execution if there is a running system-wide execution (id: {0})" ?